wrapper/src/main/java/software/amazon/jdbc/wrapper/ConnectionWrapper.java

```diff
@@ -99,10 +101,17 @@ public ConnectionWrapper(
             effectiveConnectionProvider,
             this,
             telemetryFactory);
+    final StorageService storageService = new StorageServiceImpl();
```

Comment:
Do we to initialize a new copy of StorageService? As I see it's provided to the constructor.